bps/clustering/clustering_ApTemplate.yaml

```diff
@@ -12,8 +12,7 @@
 clusterAlgorithm: lsst.ctrl.bps.quantum_clustering_funcs.dimension_clustering
 cluster:
   singleFrame:
-    # TODO DM-40389: replace this with isr,calibrateImage
-    pipetasks: isr,characterizeImage,calibrate
+    pipetasks: isr,calibrateImage
```

Comment:
Leave this all exactly as is.

Reply:
`ApTemplate` [does, in fact, use `calibrateImage`](https://github.com/lsst/ap_pipe/blob/main/pipelines/_ingredients/ApTemplate.yaml#L12).

Reply:
Oh, hmm, I was not intending on changing that, but now I see that the import there made it change.

Hopefully that's all moot, as we should be moving away from ApTemplate to just using the DRP pipeline.